### PR TITLE
added Sentry integration

### PR DESCRIPTION
**Pull request by vesellov**

### CODE REVIEW

File: src/main/settings.py [hunk 1]
```diff
@@ -24,6 +24,21 @@
 else:
     from main import params
 
+
+#------------------------------------------------------------------------------
+#--- Sentry config
+SENTRY_ENABLED = getattr(params, 'SENTRY_ENABLED', False)
```
Comment (ilhanmercan) [inline src/main/settings.py:30]: we should have these settings in params example file

Comment (vesellov) [reply src/main/settings.py:30]: can you pls look more carefully @ilhanmercan

File: src/main/params_example.py [hunk 2]
```diff
@@ -13,6 +13,10 @@
 import os
 os.umask(0o002)
 
+#--- Sentry
+# SENTRY_ENABLED = False
+# SENTRY_DSN = ''
+
```
Comment (vesellov) [inline src/main/params_example.py:19]: those were already there

File: src/main/settings.py (hunk 1 above)
Comment (ilhanmercan) [reply src/main/settings.py:30]: ohh, you are right. somehow I missed it :(

**Review by ilhanmercan — APPROVED** [review]

**Pull request merged**